### ISSUE TITLE
how to play GAIL result in mujoco

### ISSUE THREAD

**Issue by huangjiancong1** [openai/baselines#501]
Hi, I want to know that how to play the result train from GAIL in mujoco. I have finish training and i want to visualize it in mujoco. Can you share me some method?

**Comment by flaurida** [comment]
I am interested in this as well - can get it to run in evaluate mode and print the return, but would be cool to see the hopper in action!

**Comment by Kailiangdong** [comment]
Any progress? I am also interesting about that. Do GAIL code have this function? Or we should develop it by ourselves, Thx

**Comment by Weiyi-Zhang258** [comment]
> I am interested in this as well - can get it to run in evaluate mode and print the return, but would be cool to see the hopper in action!

How to run in evaluate mode? I don't know what args should I write.
When I run in train mode, the model is saved by tf.train.saver
But when I run in evaluate mode, it would read the model by "U.load_variables(load_model_path)", how should I set the arg to read in the model?

**Comment by shibei00** [comment]
I have the same question...

**Comment by shibei00** [comment]
> > I am interested in this as well - can get it to run in evaluate mode and print the return, but would be cool to see the hopper in action!
> 
> How to run in evaluate mode? I don't know what args should I write.
> When I run in train mode, the model is saved by tf.train.saver
> But when I run in evaluate mode, it would read the model by "U.load_variables(load_model_path)", how should I set the arg to read in the model?

Use "--task evaluate"

**Comment by Gaoee** [comment]
> > > I am interested in this as well - can get it to run in evaluate mode and print the return, but would be cool to see the hopper in action!
> > 
> > 
> > How to run in evaluate mode? I don't know what args should I write.
> > When I run in train mode, the model is saved by tf.train.saver
> > But when I run in evaluate mode, it would read the model by "U.load_variables(load_model_path)", how should I set the arg to read in the model?
> 
> Use "--task evaluate"

I have tried this argument, but it still has to get the model path. Is there anyone has fixed this problem?

**Comment by shibei00** [comment]
You can use "--load_model_path" to specify your model path. Your model is a *.variables file.

**Comment by shibei00** [comment]
> > > > I am interested in this as well - can get it to run in evaluate mode and print the return, but would be cool to see the hopper in action!
> > > 
> > > 
> > > How to run in evaluate mode? I don't know what args should I write.
> > > When I run in train mode, the model is saved by tf.train.saver
> > > But when I run in evaluate mode, it would read the model by "U.load_variables(load_model_path)", how should I set the arg to read in the model?
> > 
> > 
> > Use "--task evaluate"
> 
> I have tried this argument, but it still has to get the model path. Is there anyone has fixed this problem?

You can use "--load_model_path" to specify your model path. Your model is a *.variables file.

**Comment by Kailiangdong** [comment]
@Gaoee Do you solve that? I have tried different path from checkpoint
